src/table.rs

```diff
@@ -292,26 +368,112 @@ impl Table {
         Table::try_new(new_schema, new_record_batches)
     }
 
+    /// Returns the length of this table.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #[cfg(feature = "geozero")]
+    /// # {
+    /// use std::fs::File;
+    ///
+    /// let file = File::open("fixtures/roads.geojson").unwrap();
+    /// let table = geoarrow::io::geojson::read_geojson(file, Default::default()).unwrap();
+    /// assert_eq!(table.len(), 21);
+    /// # }
+    /// ```
     pub fn len(&self) -> usize {
         self.batches.iter().fold(0, |sum, val| sum + val.num_rows())
     }
 
+    /// Returns true if this table is empty.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #[cfg(feature = "geozero")]
+    /// # {
+    /// use std::fs::File;
+    ///
+    /// let file = File::open("fixtures/roads.geojson").unwrap();
+    /// let table = geoarrow::io::geojson::read_geojson(file, Default::default()).unwrap();
+    /// assert!(!table.is_empty());
+    /// # }
+    /// ```
     pub fn is_empty(&self) -> bool {
         self.len() == 0
     }
 
+    /// Consumes this table, returning its schema and its record batches.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #[cfg(feature = "geozero")]
+    /// # {
+    /// use std::fs::File;
+    ///
+    /// let file = File::open("fixtures/roads.geojson").unwrap();
+    /// let table = geoarrow::io::geojson::read_geojson(file, Default::default()).unwrap();
+    /// let (schema, record_batches) = table.into_inner();
+    /// # }
+    /// ```
     pub fn into_inner(self) -> (SchemaRef, Vec<RecordBatch>) {
         (self.schema, self.batches)
     }
```

Comment:
Here we should probably change this ordering as well to `(batches, schema)` to be consistent.